misc/install-xattr/install-xattr.c

```diff
@@ -383,6 +386,7 @@ main(int argc, char* argv[])
 			if (!opts_target_directory) {
 				target = argv_copy[last];
 				if (stat(target, &s) != 0) {
+					free(mypath);
```

Comment:
Why not free it immediately after `default:`? it doesn't seem to be used elsewhere.

In fact, given it's only used in one place, perhaps the whole variable can be scoped in `case 0:`. Also, the program is completely missing error handling for it.

Reply:
Oh, good spot.

Reply:
You seem to have implemented my first paragraph but not the second. Any reason not to?

Reply:
Just no time yet - that's why the comment isn't resolved.